sql_metadb/derived_tables/users_departments_unpacked.sql

```diff
@@ -0,0 +1,52 @@
+DROP TABLE IF EXISTS users_departments_unpacked;
+
+-- Create a derived table that takes the users table and joins
+-- in the department information
+CREATE TABLE users_departments_unpacked AS
+WITH departments_array AS (
+    SELECT
+        uu.id::uuid AS user_id,
```

Comment:
`uu.id` is already type UUID but it does not hurt it being here.

Reply:
Agree with @lib-eric